packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -42,6 +42,27 @@ export interface InstanceProps {
    * @default no parameter group
    */
   readonly parameterGroup?: IParameterGroup;
+
+  /**
+   * Whether to enable Performance Insights for the DB instance.
+   *
+   * @default false
+   */
+  readonly enablePerformanceInsights?: boolean;
+
+  /**
+   * The amount of time, in days, to retain Performance Insights data.
+   *
+   * @default 7
+   */
+  readonly performanceInsightRetention?: PerformanceInsightRetention;
+
+  /**
+   * The AWS KMS key for encryption of Performance Insights data.
+   *
+   * @default - default master key
+   */
+  readonly performanceInsightEncryptionKey?: kms.IKey;
```

Comment:
Ditto - we should mention here that this is ignored currently unless `enablePerformanceInsights` is `true`.